### PR TITLE
docs: add Overwolf to showcase

### PR DESCRIPTION
**Pull request by eransharv**

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -900,6 +900,14 @@ const Users: User[] = [
     source: 'https://github.com/orbitjs/orbit/tree/main/website',
     tags: ['opensource', 'versioning'],
   },
+  {
+    title: 'Overwolf',
+    description: 'Easily create apps for PC gamers',
+    preview: require('./showcase/overwolf.png'),
+    website: 'https://www.overwolf.com/',
+    source: 'https://overwolf.github.io',
```
Comment (armano2) [inline website/src/data/users.tsx:908]: i'm not 100% sure but shouldn't this be

```suggestion
    website: 'https://overwolf.github.io',
    source: 'https://github.com/overwolf/overwolf.github.io',
```

Comment (eransharv) [reply website/src/data/users.tsx:908]: @armano2 hey. these are the correct URLs:

Our main product site is: https://www.overwolf.com/
And our documentation site for devs is: https://overwolf.github.io

Comment (Josh-Cena) [reply website/src/data/users.tsx:908]: @eransharv Armano is correct. `website` means your deployed site, and `source` means a link to a GitHub repo or anywhere that people can inspect your site's source code.

Comment (eransharv) [reply website/src/data/users.tsx]: @Josh-Cena oh so sorry.. fixed it. Thanks for the clarifications.

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -900,6 +900,14 @@ const Users: User[] = [
     source: 'https://github.com/orbitjs/orbit/tree/main/website',
     tags: ['opensource', 'versioning'],
   },
+  {
+    title: 'Overwolf',
+    description: 'Easily create apps for PC gamers',
+    preview: require('./showcase/overwolf.png'),
+    website: 'https://overwolf.github.io',
+    source: 'https://github.com/overwolf/overwolf.github.io',
+    tags: ['product', 'large'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:909]:
```suggestion
    tags: ['opensource', 'product', 'large'],
```

File: website/src/data/users.tsx [hunk 3]
```diff
@@ -900,6 +900,14 @@ const Users: User[] = [
     source: 'https://github.com/orbitjs/orbit/tree/main/website',
     tags: ['opensource', 'versioning'],
   },
+  {
+    title: 'Overwolf',
+    description: 'Easily create apps for PC gamers',
+    preview: require('./showcase/overwolf.png'),
+    website: 'https://overwolf.github.io',
+    source: 'https://github.com/overwolf/overwolf.github.io',
+    tags: ['opensource', 'product', 'large'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:909]:
```suggestion
    tags: ['favorite', 'opensource', 'design', 'product', 'large'],
```

**Comment by Josh-Cena** [comment]
Oops, just realized that your site is built with Docusaurus v1 😅 Despite it being very well-designed, our showcase is exclusively for sites built with v2. Please re-send a PR targeting the v1 branch as it's quite intractable to switch targets.

**Pull request closed without merging**